jsettlers.main.swing/src/jsettlers/main/swing/SwingManagedJSettlers.java

```diff
@@ -79,6 +81,44 @@ public static void main(String[] args) throws FileNotFoundException, IOException
 		generateContent(argsMap, content);
 	}
 
+	private static String getSelectedFolderName() {
+		
+		//- Info: Don't use JFileChooser(), some Windows User have problems caused by a NullPointerException in FileDialog()
+		//-   see issue #240: https://github.com/jsettlers/settlers-remake/issues/240
+		
+		
+		//- see: http://stackoverflow.com/questions/1224714/how-can-i-make-a-java-filedialog-accept-directories-as-its-filetype-in-os-x
+		JFrame frame = new JFrame();
+		
+		System.setProperty("apple.awt.fileDialogForDirectories", "true");
```

Comment:
I have wrote a directory chooser in Swing, which looks similar to the Windows one.
But I have to remove some dependencies, then it can be copied, because I will not use the whole library...
Should I prepare it?